platforms/chibios/boards/common/ld/stm32f4xx_common.ld

```diff
@@ -24,12 +24,12 @@ MEMORY
     flash5 (rx) : org = 0x00000000, len = 0
     flash6 (rx) : org = 0x00000000, len = 0
     flash7 (rx) : org = 0x00000000, len = 0
-    ram0   (wx) : org = 0x20000000, len = f4xx_ram_size
-    ram1   (wx) : org = 0x00000000, len = 0
-    ram2   (wx) : org = 0x00000000, len = 0
+    ram0   (wx) : org = 0x20000000, len = f4xx_ram1_size + f4xx_ram2_size /* SRAM1 + SRAM2 */
+    ram1   (wx) : org = 0x20000000, len = f4xx_ram1_size                  /* SRAM1 */
+    ram2   (wx) : org = 0x2001C000, len = f4xx_ram2_size                  /* SRAM2 */
```

Comment:
Some of the math didn't look right before.  But that may be because some of it was wrong.  Double checking, this looks good and a better way to handle this. 

And yeah, I saw the SRAM3 setting on a couple of the controllers.  But yeah, figured it would wait until we have support for those, as they'd also need the board config.